### PR TITLE
docs: update readme to use paystring

### PR DESCRIPTION
**Pull request by dino-rodriguez**
## High Level Overview of Change

Update README to use PayString.

### Context of Change

PayString rebrand.

### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [x] Documentation Updates
- [ ] Release

## Before / After

PayID -> PayString

## Test Plan

None

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,25 @@
...
-[![Public API Version: 1.0](https://img.shields.io/badge/Public%20API%20Version-1.0-blue)](https://github.com/payid-org/payid/blob/master/src/config.ts#L1)
-[![Admin API Version: 2020-05-28](https://img.shields.io/badge/Admin%20API%20Version-2020--05--28-blue)](https://github.com/payid-org/payid/blob/master/src/config.ts#L2)
+[![License: Apache 2](https://img.shields.io/badge/license-Apache%202-brightgreen)](https://github.com/PayString/paystring/blob/master/LICENSE)
+[![Public API Version: 1.1](https://img.shields.io/badge/Public%20API%20Version-1.1-blue)](https://github.com/PayString/paystring/blob/master/src/config.ts#L1)
+[![Admin API Version: 2020-08-28](https://img.shields.io/badge/Admin%20API%20Version-2020--08--25-blue)](https://github.com/PayString/paystring/blob/master/src/config.ts#L2)
 
-Welcome to PayID, the universal payment identifier.
+Welcome to PayString, the universal payment identifier.
 
-_This project is not associated with PayID operated by NPP Australia Ltd. People in Australia are prohibited from using this project. See below for more details._
```
Comment (dino-rodriguez) [inline README.md:9]: @Stormtv I'm assuming we can remove this if we are rebranding as `PayString` now?

File: README.md [hunk 2]
```diff
@@ -1,27 +1,25 @@
...
-- [PayID Developer Documentation](https://docs.payid.org/payid-overview)
-- [PayID API Documentation](https://api.payid.org/?version=latest)
-- [Core PayID Protocol RFCs](https://github.com/payid-org/rfcs)
+- [PayString Developer Documentation](https://docs.paystring.org/getting-started)
+- [PayString API Documentation](https://api.paystring.org/?version=latest)
+- [Core PayString Protocol RFCs](https://github.com/PayString/rfcs)
 
 ## Legal
 
-By using, reproducing, or distributing this code, you agree to the terms and conditions for use (including the Limitation of Liability) in the [Apache License 2.0](https://github.com/payid-org/payid/blob/master/LICENSE). If you do not agree, you may not use, reproduce, or distribute the code. **This code is not authorised for download in Australia. Any persons located in Australia are expressly prohibited from downloading, using, reproducing or distributing the code.** This code is not owned by, or associated with, NPP Australia Limited, and has no sponsorship, affiliation or other connection with the “Pay ID” service operated by NPP Australia Limited in Australia.
```
Comment (dino-rodriguez) [inline README.md:27]: Same here.

File: README.md (hunk 1 above)
Comment (Stormtv) [reply README.md:9]: I'd double check but my opinion would be this is okay to drop since paystring.org is available in Australia

**Review by Stormtv — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (dino-rodriguez) [reply README.md:9]: @Stormtv Following up from our slack discussion -- deciding to keep this in until we cut a major version release to swap out the code changes ( which includes the version header ).

**Review by austin-king — APPROVED** [review]
LGTM

**Pull request merged**